scripts/gitlab/build_and_test.sh

```diff
@@ -253,7 +253,15 @@ then
     # make clean
 fi
 
+cd ${project_dir}
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
 echo "~~~~~ Build and test completed"
 echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+echo "~~~~~ To reproduce this build:"
+echo ""
+echo " #`git rev-parse HEAD` on host ${truehostname}"
```

Comment:
Thanks @davidbeckingsale - This looks great!  Would you mind changing the following 3 lines to:
```suggestion
echo "On host ${truehostname}, run:"
echo ""
echo "git checkout `git rev-parse HEAD` && git submodule init && git submodule update"
echo "SPACK_DISABLE_LOCAL_CONFIG=\"\" SPACK_USER_CACHE_PATH=\"../ci_spack_cache\" python3 scripts/uberenv/uberenv.py \"${spec}\" --prefix=\"../ci_uberenv\'""
echo "mkdir build && cd build"
echo "cmake .. -C ../${truehostname}-*.cmake"
echo "make && make test
```

Reply:
This would make the output look something like:
```
On host corona, run:

git checkout fe91815d1850310ec894e2d2eae1d6b89cabafca && git submodule init && git submodule update
SPACK_DISABLE_LOCAL_CONFIG="" SPACK_USER_CACHE_PATH="../ci_spack_cache" python3 scripts/uberenv/uberenv.py "~shared +fortran +device_alloc tests=basic  +rocm amdgpu_target=gfx906 %rocmcc@5.5.0 ^hip@5.5.0 " --prefix="../ci_uberenv"
mkdir build && cd build
cmake .. -C ../corona-\*.cmake
make && make test
```